### PR TITLE
Upgrading dependencies and migrating to new Lab and code

### PR DESCRIPTION
**Pull request by singhs020**

### CODE REVIEW

**Comment by midknight41** [comment]
@jmac105 any concerns from you?

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "7"
-  - "6"
-  - "4"
+  - "8"
+  - "10"
```
Comment (jmac105) [inline .travis.yml:4]: By the end of today Node 12 will be LTS, worth adding now? Agree with adding even numbers only

**Review by jmac105 — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (singhs020) [reply .travis.yml:4]: I am waiting for a confirmed release number to give it a test on my local before raising a PR.

Comment (singhs020) [reply .travis.yml:4]: Added node v12 in travis config

**Comment by jmac105** [comment]
Thanks @simranjeet2589 , sorry they have taken so long to publish!!

**Pull request merged**